src/community/release/ext-features-templating.xml

```diff
@@ -10,8 +10,8 @@
       <outputDirectory></outputDirectory>
       <includes>
         <include>gs-features-templating*.jar</include>
-	<include>jsonld-java-*.jar</include>
-	<include>gt-geojson-*.jar</include>
+        <include>jsonld-java-*.jar</include>
+        <include>gt-geojson-*.jar</include>
```

Comment:
This module is being deprecated and will be replaced by gt-geojsonstore (which also provides JSON parsing utilities outside of the usage of a DataStore).  Maybe some parts of GeoServer you're depending onto still use it?

Reply:
fixed